src/tree/registries/registryTreeUtils.ts

```diff
@@ -17,33 +17,35 @@ export function getImageNameFromRegistryTagItem(tag: CommonTag): string {
     return `${repository.label.toLowerCase()}:${tag.label.toLowerCase()}`;
 }
 
-export function getBaseUrlFromItem(item: CommonRegistryItem): string {
-    if (!isTag(item) && !isRepository(item) && !isRegistry(item)) {
-        throw new Error(l10n.t('Unable to get base URL'));
+export function getBaseImagePathFromRegistry(registry: CommonRegistry): string {
+    if (!isRegistry(registry)) {
+        throw new Error(l10n.t('Unable to get base image path'));
+    } else if (isAzureRegistry(registry) || isGenericV2Registry(registry) || isGitHubRegistry(registry)) {
+        return registry.baseUrl.authority.toLowerCase();
+    } else if (isDockerHubRegistry(registry)) {
```

Comment:
This will make it impossible to work for any other registry providers (including GitLab). We should still have an `else` case in this if/else block.

Reply:
yeah you're right, i'm thinking we should do `registry.baseUrl.authority.toLowerCase();` for the else case